docs/libcudacxx/extended_api/synchronization_primitives/atomic_ref.rst

```diff
@@ -38,13 +38,17 @@ with the following additional operations. This class additionally deviates from
 Limitations
 -----------
 
-``cuda::atomic_ref<T>`` and ``cuda::std::atomic_ref<T>`` may only be instantiated with a T that are either 4 or 8 bytes.
+``cuda::atomic_ref<T>`` and ``cuda::std::atomic_ref<T>`` may only be instantiated when ``T`` satisfies ``sizeof(T) <= 16``.
+
+The operations available to ``T`` when ``sizeof(T) == 16`` depend on the architecture:
+  - On SM70 and later: ``load`` and ``store`` are supported.
+  - On SM90 and later: ``fetch_*`` and synchronization operations are supported, implemented via CAS.
 
 No object or subobject of an object referenced by an ``atomic_­ref`` shall be concurrently referenced by any other
 ``atomic_­ref`` that has a different ``Scope``.
 
 For ``cuda::atomic_ref<T>`` and ``cuda::std::atomic_ref<T>`` the type ``T`` must satisfy the following:
-  - ``sizeof(T) <= 8``.
+  - ``sizeof(T) <= 8 // (16 for SM70+)``.
```

Comment:
I find it confusing that we specify this differently than above. We should use the same description

Reply:
I think this is resolved.